### DISCUSSION TITLE
Is Herd (Pro) useful/recommended for WordPress development?

### DISCUSSION THREAD

**Discussion by brigleb** [beyondcode/herd-community#560] [Q&A]
I’ve been trying Herd Pro out, but I’m struggling to get a lot of basic WordPress things working. Some examples off the top of my head:

- Setting up new sites requires manual steps setting up the database that are outside the scope of Herd Pro’s Services feature.
- The log files WordPress generates don’t seem to be available to Herd Pro.
- I can’t seem to get Xdebug to work, though that may very well be an issue on my end.

Is anyone else using this for WordPress development? Should it be useful for WordPress development? I certainly like how clean and speedy it is, but if it’s the wrong tool for the job, I should probably change gears before spending too much time on this. Thoughts?

**Comment by maxcelos** [comment]
I'm using it currently for simple WP development and works fine to me. I don't really need Xdebug or extensive use of WP logs, it is not holding me on.

Before Herd I was using LocalWP, that is very handy, but kinda overwelming to me. Also consumes too much from my machine. The only downside so far is lose the connectivit to push/pull with WPEngine. But I also don't use it frequently. This way it not a big deal for me.

I've implemented a simple node script that uses wp-cli to create the installation inside a parked folder, and database, just by providing a folder name. With a simple command line, I have a https clean new wp site working all the time with database available in a single mysql server for all sites, instead of having all separated in LocalWP and needing to start/stop on demand.

Would be awesome to have this creation workflow inside the Herd.

**Comment by vinkla** [comment]
We are developing WordPress websites using [WordPlate](http://github.com/vinkla/wordplate), which is installed with Composer. We have been able to utilize many features in Herd, such as mail debugging and logging dumps in Herd Pro.

**Comment by brigleb** [comment]
Thank you both for the insight! I am coming from LocalWP, and I'm slowly getting used to Herd at this point. I do wish, as @maxcelos mentions, that there was more of a helpful setup workflow for WordPress in Herd, as there is for Laravel, and I will look into WordPlate as @vinkla suggests, which looks promising.

**Comment by LukaszJaro** [comment]
Here's the basics.
https://www.briancoords.com/local-wordpress-with-herd-dbngin-and-wp-cli/

Additionally I added Mailpit to capture mail, which was very simple.

Herd replaced Laragon for me which has gone the paid route and it has been very good, faster even. To me pro version would automate alot more things, that's probably the one main benefit. For example, now I need run command to start mail service, I need to open DBngin and start services. Not huge deal, depends how often you restart your machine.
